packages/ckeditor5-media-embed/src/ui/mediaformview.ts

```diff
@@ -342,11 +319,11 @@ export default class MediaFormView extends View {
  * Fired when the form view is submitted (when one of the children triggered the submit event),
  * e.g. click on {@link #saveButtonView}.
  *
- * @event submit
+ * @eventName submit
```

Comment:
`eventName` should be attached to a type describing this event. It seems not used in this code anyway.